pxr/usdImaging/bin/testusdview/testenv/testUsdviewPrimTreeExpansion/testUsdviewPrimTreeExpansion.py

```diff
@@ -128,6 +128,19 @@ def _testAllExpanded(appController):
     _selectVariant(appController, CAPSULE[VARIANT_INFO_POS], FIRST_VARIANT)
     _expandPrims(appController, ["/spheres", "/A", "/A/B", "/A/B/C"])
     initialExpandedPrims = _getExpandedPrims(appController)
+
+    # test to see if the display name of the capsule is showing
+    prim = appController._dataModel.stage.GetPrimAtPath("/Shapes/Pill")
+    item = appController._primToItemMap.get(prim)
+    assert item._nameData(QtCore.Qt.DisplayRole) == "CapsuleDisplayName"
+
+    # clear the a-variant and just select the b-variant
+    _selectVariant(appController, 0, FIRST_VARIANT)
```

Comment:
It's confusing to see a zero, here, and since no such variantSet was being set previously, makes me just a tad concerned that the clause below your new code is getting its initial conditions whisked out from under it?

Can we add a tuple up top like we do for the other VariantSets so it's more consistent here?